src/plugins/data/public/query/timefilter/timefilter_service.ts

```diff
@@ -26,9 +32,10 @@ export class TimefilterService {
   constructor(private readonly nowProvider: NowProviderInternalContract) {}
 
   public setup({ uiSettings, storage }: TimeFilterServiceDependencies): TimefilterSetup {
-    const timefilterConfig = {
+    const timefilterConfig: TimefilterConfig = {
       timeDefaults: uiSettings.get(UI_SETTINGS.TIMEPICKER_TIME_DEFAULTS),
       refreshIntervalDefaults: uiSettings.get(UI_SETTINGS.TIMEPICKER_REFRESH_INTERVAL_DEFAULTS),
+      minRefreshIntervalDefault: 5000,
```

Comment:
I don't know where to pull the value from to be serverless specific. Should this live in the [`data` plugin config](https://github.com/elastic/kibana/blob/main/src/plugins/data/config.ts)?